web_external/views/body/SubmitView.js

```diff
@@ -72,9 +86,17 @@ var SubmitView = View.extend({
         this.listenTo(this.uploadWidget, 'g:filesChanged', this.filesSelected);
         this.listenTo(this.uploadWidget, 'g:uploadStarted', this.uploadStarted);
         this.listenTo(this.uploadWidget, 'g:uploadFinished', this.uploadFinished);
+        this.$('.c-submission-approach-input').typeahead({
```

Comment:
The input element's content is updated when an entry is selected from the dropdown, but `this.approach` is not set. This results in unexpected behavior when submitting.

It looks like installing an `afterSelect` handler on the typeahead to set `this.approach` could fix this.